assets/models/organelles/Nucleus.tscn

```diff
@@ -1,69 +1,27 @@
 
-[sub_resource type="ShaderMaterial" id="1"]
+[sub_resource type="ShaderMaterial" id="ShaderMaterial_ngmrq"]
 resource_local_to_scene = true
 render_priority = 0
-shader = ExtResource("2")
+shader = ExtResource("1_bheag")
 shader_parameter/dissolveValue = 0.0
-shader_parameter/fresnelValue = 1.0
+shader_parameter/dissolveScale = 1.8
+shader_parameter/fresnelValue = 0.0
 shader_parameter/fresnelActive = false
-shader_parameter/outlineWidth = 0.05
-shader_parameter/growColor = Color(0.431373, 0.431373, 0.431373, 1)
+shader_parameter/outlineWidth = 0.1
+shader_parameter/growColor = Color(0.176419, 0.176419, 0.176419, 1)
 shader_parameter/tint = Color(1, 1, 1, 1)
-shader_parameter/jiggleAmount = 0.0005
-shader_parameter/texture = ExtResource("8")
-shader_parameter/dissolveTexture = ExtResource("9")
-
-[sub_resource type="ShaderMaterial" id="2"]
-resource_local_to_scene = true
-render_priority = -1
-shader = ExtResource("2")
-shader_parameter/dissolveValue = 0.0
-shader_parameter/fresnelValue = 1.0
-shader_parameter/fresnelActive = false
-shader_parameter/outlineWidth = 0.05
-shader_parameter/growColor = Color(0.362829, 0.362829, 0.362829, 1)
-shader_parameter/tint = Color(1, 1, 1, 1)
-shader_parameter/jiggleAmount = 0.0005
-shader_parameter/texture = ExtResource("6")
-shader_parameter/dissolveTexture = ExtResource("9")
-
-[sub_resource type="ShaderMaterial" id="3"]
-resource_local_to_scene = true
-render_priority = -1
-shader = ExtResource("2")
-shader_parameter/dissolveValue = 0.0
-shader_parameter/fresnelValue = 1.0
-shader_parameter/fresnelActive = false
-shader_parameter/outlineWidth = 0.05
-shader_parameter/growColor = Color(0.409176, 0.452529, 0.462167, 1)
-shader_parameter/tint = Color(1, 1, 1, 1)
-shader_parameter/jiggleAmount = 0.0005
-shader_parameter/texture = ExtResource("5")
-shader_parameter/dissolveTexture = ExtResource("9")
+shader_parameter/jiggleAmount = 0.01
+shader_parameter/texture = ExtResource("3_1w7t3")
+shader_parameter/dissolveTexture = ExtResource("2_tdl0s")
 
 [node name="Nucleus" type="MeshInstance3D"]
-transform = Transform3D(127.947, 0, 0, 0, -2.08452e-05, 127.947, 0, -127.947, -2.08452e-05, 0, 0, 0)
-material_override = SubResource("1")
-mesh = ExtResource("1")
-script = ExtResource("3")
-
-[node name="Golgi" type="MeshInstance3D" parent="."]
-transform = Transform3D(-0.74586, -0.666102, -1.08522e-07, 0.666102, -0.74586, -2.84437e-07, 1.08522e-07, -2.84437e-07, 0.999999, -0.00490416, -0.0155195, -2.52845e-09)
-material_override = SubResource("2")
-mesh = ExtResource("4")
-
-[node name="ER" type="MeshInstance3D" parent="."]
-transform = Transform3D(-0.630648, 0.776067, 1.26437e-07, -0.776067, -0.630648, -2.65666e-07, -1.26437e-07, -2.65666e-07, 0.999999, 0.0103581, -0.0146696, -0.00807817)
-material_override = SubResource("3")
-mesh = ExtResource("7")
+rotation_edit_mode = 2
+material_override = SubResource("ShaderMaterial_ngmrq")
+mesh = ExtResource("4_tmwr2")
+skeleton = NodePath("")
```

Comment:
Skeleton? Why does the Nucleus have a skeleton set.

Reply:
It doesn't have a skeleton. I'm not sure how that got added.